lib/Dialect/TritonGPU/Transforms/Pipeliner/MatmulLoopPipeline.cpp

```diff
@@ -0,0 +1,691 @@
+#include "PipelineExpander.h"
```

Comment:
I suppose AMD could have an independent file for pipelining and set up proper pipelining options?

Reply:
Yes that's the idea. I think we should also be able to extra common helpers to make it easier to create schedules but I don't have a clear idea of which helpers would be useful at this point.